projects/ngx-formentry/src/form-entry/helpers/js-expression-helper.ts

```diff
@@ -286,6 +286,15 @@ export class JsExpressionHelper {
     return !!targetControl ? targetControl : typeof obsValue === 'object' ? obsValue.uuid : !!obsValue ? obsValue : null
   }
 
+  evaluateMatchExpression(regexString, val) {
+    if (!val || ['undefined', 'null', ''].includes(val.toString())) {
+      return true;
+    }
+  
+    const regex = new RegExp(regexString);
+  return !regex.test(val);
+  }
```

Comment:
We could improve this function by naming it `isMatch` and having it return true if the `value` matches the `regexString`.  Then use the returned value to achieve the intended behavior within the form.